README.LOCAL-ANALYSIS.md

```diff
@@ -161,3 +161,27 @@ Each analysis run takes up a significant amount of limited VM disk space. To cle
 ```
 ./scripts/clean-analysis-volumes
 ```
+
+## Importing local analysis results into the BNA site
+
+The results of analysis jobs run locally can be imported and displayed on the [BNA site](https://bna.peopleforbikes.org/).
+
+To do so, first [create a Neighborhood](https://bna.peopleforbikes.org/#/admin/neighborhoods/create/)
+(if it doesn't already exist on the site) using the boundary shapefile you used for the analysis.
+Then go to the ["Import Analysis Results" page](https://bna.peopleforbikes.org/#/admin/analysis-jobs/import/),
+select the neighborhood, and provide a URL to a zip file containing the local analysis results.
+
+The results zip file should contain all the files written to the `NB_OUTPUT_DIR` directory by the
```

Comment:
It's also possible to collect these files by downloading them from another instance. There is one file (a Shapefile, I think) that is in the local output directory but not available for download, and is not required for re-importing.